submissions/team_api.py

```diff
@@ -234,19 +234,20 @@ def get_teammates_with_submissions_from_other_teams(
     Returns:
         list(dict): [{ 'student_id', 'team_id' }]
     """
-    items = StudentItem.objects.filter(
-        submission__team_submission__isnull=False
+    submissions = Submission.objects.filter(
+        student_item__student_id__in=team_member_ids,
+        student_item__course_id=course_id,
+        student_item__item_id=item_id,
     ).exclude(
-        submission__team_submission__team_id=team_id
-    ).filter(
-        student_id__in=team_member_ids,
-        course_id=course_id,
-        item_id=item_id
-    ).values("student_id", "submission__team_submission__team_id")
+        team_submission__isnull=True,
+    ).exclude(
+        team_submission__team_id=team_id,
```

Comment:
we don't need to actually specify we're excluding DELETED because the submissions manager handles that for us.